### DISCUSSION TITLE
Allure report not displayed Failed status screenshot

### DISCUSSION THREAD

**Discussion by RaghvueerNayak** [allure-framework/allure2#2266] [Questions & Support]
Allure Report : Not displayed correct status and screenshots
Whiile running test cases sucessfully with failed status in eclipse console. I found no screenshot attached to Test Body( Allure report) 
My BaseClass java file trigered both way ,one with stored screenshots to screenshot folder and another one allure report. Here No issue with first part but second , having issue with screen shot.( 
[allure query.pdf](https://github.com/allure-framework/allure2/files/13751360/allure.query.pdf)

it captured all required data in Allure-Report folder but not dispalyed correct status and screenshots in Browser (Chrome) (Attached screen shots)
Please help me why screen shots and Correct status not captured in Allure report ( Chrome Browser)

**Comment by baev** [comment]
All the issue details & working examples can be found here https://github.com/biczomate/allure-testng7.5-attachment-example
